hub/instruction_handling/storage/constraints.tex

```diff
@@ -190,6 +197,13 @@
 						% {i}{\locSecondStoRowOffset}{0} \\
 					\end{array} \right.
 				\]
+				We further impose, again,
+				\[
+					\left\{ \begin{array}{lcl}
+						\stoSloadOperation  _{i + \locSecondStoRowOffset} & = & \locIsSload  \\
+						\stoSstoreOperation _{i + \locSecondStoRowOffset} & = & \locIsSstore \\
+					\end{array} \right.
+				\]
```

Comment:
We set `storage/SSTORE_OPERATION` and `storage/SLOAD_OPERATION` only for storage instructions. They are implicitly set to 0 along pre-warming rows due to flag exclusivity.